### DISCUSSION TITLE
GUI Not appering in Pychram 3.11

### DISCUSSION THREAD

**Discussion by charle01** [community/community#60110] [Copilot Conversations]
### Select Topic Area

General

### Body

HI!
  I am creating a GUI based window in Python using phycharm editor .After successfully execution's of program GUI based window is not appearing on the other hand It's working on phycharm IDE platform. What could be possible root cause. pls  suggest.

**Comment by sohaibaumer** [comment]
The GUI window might not be appearing outside of PyCharm due to differences in the execution environment. Ensure that your code is not dependent on PyCharm-specific features or configurations. Verify that dependencies and libraries are installed correctly. Try running the program outside of PyCharm to see if the window appears. Double-check your code to ensure you are properly creating and displaying the GUI window.
